src/Aspire.Dashboard/Components/Resize/BrowserDimensionWatcher.razor.cs

```diff
@@ -39,27 +39,24 @@ public async Task OnResizeAsync(ViewportSize viewportSize)
     {
         var newViewportInformation = GetViewportInformation(viewportSize);
 
-        if (newViewportInformation.IsDesktop != ViewportInformation!.IsDesktop || newViewportInformation.IsUltraLowHeight != ViewportInformation.IsUltraLowHeight)
+        if (newViewportInformation.IsDesktop != ViewportInformation!.IsDesktop
+            || newViewportInformation.IsUltraLowHeight != ViewportInformation.IsUltraLowHeight
+            || newViewportInformation.IsUltraLowWidth != ViewportInformation.IsUltraLowWidth)
         {
             ViewportInformation = newViewportInformation;
             DimensionManager.IsResizing = true;
-
+            DimensionManager.InvokeOnBrowserDimensionsChanged(newViewportInformation);
             await ViewportInformationChanged.InvokeAsync(newViewportInformation);
-            DimensionManager.InvokeOnBrowserDimensionsChanged();
-
             DimensionManager.IsResizing = false;
         }
     }
 
     private static ViewportInformation GetViewportInformation(ViewportSize viewportSize)
     {
-        return new ViewportInformation(IsDesktop: viewportSize.Width > 768, IsUltraLowHeight: viewportSize.Height < 400);
-    }
-
-    public static ViewportInformation Create(int height, int width)
-    {
-        return new ViewportInformation(IsDesktop: width > 768, IsUltraLowHeight: height < 400);
+        return new ViewportInformation(
+            IsDesktop: viewportSize.Width > 768,
+            IsUltraLowHeight: viewportSize.Height < 400,
+            IsUltraLowWidth: viewportSize.Width < 350);
     }
```

Comment:
Nit: Move numbers to constants. Add comment explaining why these numbers are used